examples/hello-grudge.py

```diff
@@ -0,0 +1,117 @@
+import numpy as np
+import pyopencl as cl
+from grudge.discretization import DiscretizationCollection
+import grudge.op as op
+from meshmode.mesh.generation import generate_box_mesh
+from meshmode.array_context import PyOpenCLArrayContext
+from arraycontext import freeze, thaw
+from grudge.dof_desc import DTAG_BOUNDARY, FACE_RESTR_INTERIOR
+
+
+ctx = cl.create_some_context()
```

Comment:
Add a comment describing what this is. (I'm guessing you'd prefer to not move it under `advection`, right?)

Reply:
Added a comment [here](https://github.com/inducer/grudge/pull/152/commits/76e506d21dc1e14b25b3748658fffa0747cfe103#diff-f740fc40d4a29a0d4743d76a19fef4be2716f7bb0a647cb73466382c225193afR11). How does it look?

Reply:
Oh, sorry! I meant maybe more a (short!) comment docstring describing what this file is, for folks who don't arrive via the docs. (Maybe move the descriptive blurb from the docs?)

Reply:
Oh :). Done in https://github.com/inducer/grudge/pull/152/commits/464e5e2cea5e6c192551cdf697b29be113ebbbae#diff-f740fc40d4a29a0d4743d76a19fef4be2716f7bb0a647cb73466382c225193afR1-R10.